src/UI/Components/Diagram/components/FormModal.tsx

```diff
@@ -256,7 +256,7 @@ const FormModal = ({
             onOpenChange={(isOpen) => setIsSelectOpen(isOpen)}
             isOpen={isSelectOpen}
             onSelect={(_evt, value) => {
-              onEntityChosen(value as string, possibleForms);
+              onEntityChosen(String(value), possibleForms);
```

Comment:
so, it can never be a number, null or undefined? 
I'd expect the onEntityChosen to handle the typing instead of mutating the value here 🤔

Reply:
It can never be number, null or undefined, it was said that changing it to default(which was undefined) isn't something wanted there, so all of the select option have string value